docker_services_cli/config.py

```diff
@@ -25,20 +25,28 @@
 versions before loading a given service's version.
 """
 
-DOCKER_SERVICES_FILEPATH = "docker_services_cli/docker-services.yml"
+DOCKER_SERVICES_FILEPATH = "docker-services.yml"
```

Comment:
hmm how was it working before? or why is this needed? I'm fearful it won't work via CLI on test?

Reply:
see the other [comment](https://github.com/inveniosoftware/docker-services-cli/pull/26/files#r926655763)